recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
+  skip: True  # [not linux]
+
+requirements:
+    build:
+        - protobuf
+        - libgsasl
+        - libntlm
+        - libxml2
+        - krb5
+        - libuuid
+        - boost ==1.60
+        - cmake
+    run:
+        - protobuf
+        - libgsasl
+        - libntlm
+        - libxml2
+        - krb5
+        - libuuid
+        - boost ==1.60
+
+test:
+  commands:
+    - test -f $PREFIX/include/hdfs/hdfs.h
+    - test -f $PREFIX/lib/libhdfs3.a
+    - test -f $PREFIX/lib/libhdfs3.so
+    - test -f $PREFIX/lib/libhdfs3.so.1
+    - test -f $PREFIX/lib/pkgconfig/libhdfs3.pc
```

Comment:
No need. Just a basic test that ensure that something you expect is there is enough. I would even remove most of the lines above and check only for the static lib and/or the dyn lib.